src/api/app/views/webui2/webui/package/_tabs.html.haml

```diff
@@ -0,0 +1,51 @@
+.bg-light
```

Comment:
It's now responsive, but I would like to avoid the duplication of links (if possible) and also have the dropdown display the name of the current page (now it's just displaying `Menu`)

Reply:
Move them to a partial with *parameters*? Like 

```ruby
render partial: 'whatever', locals { link_class: 'nav-link', wrapper: '%li.nav-item' }
```


```ruby
render partial: 'whatever', locals { link_class: 'dropdown-item', wrapper: '%div' }
```